From f65ee78223ba8bee84194067fa6d35bccded86a5 Mon Sep 17 00:00:00 2001
From: Chandrakala Chavva <cchavva@caviumnetworks.com>
Date: Wed, 21 Aug 2013 22:34:30 -0700
Subject: [PATCH 221/518] MIPS: OCTEON: Fix compilation errors with previous
 revert.

Source: Cavium Networks, Inc.
MR: 00000
Type: Integration
Disposition: Merged from Octeon Tree
ChangeID: 68d224b0b9ac06cb3ff707ae2c38949bf54f5c18
Description:

Signed-off-by: Chandrakala Chavva <cchavva@caviumnetworks.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
[Original patch taken from Cavium SDK 3.1.1 525]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/mm/c-octeon.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/mm/c-octeon.c b/arch/mips/mm/c-octeon.c
index 1dba140..df1f41e 100644
--- a/arch/mips/mm/c-octeon.c
+++ b/arch/mips/mm/c-octeon.c
@@ -76,8 +76,8 @@ static void octeon_local_flush_icache_range(unsigned long start,
  */
 static void octeon_flush_icache_all_cores(struct vm_area_struct *vma)
 {
-	extern void octeon_send_ipi_single(int cpu, unsigned int action);
 #ifdef CONFIG_SMP
+	extern struct plat_smp_ops *mp_ops;	/* private */
 	int cpu;
 	cpumask_t mask;
 #endif
-- 
1.9.1

